br/pkg/restore/data.go

```diff
@@ -195,73 +211,36 @@ func (recovery *Recovery) ReadRegionMeta(ctx context.Context, totalTiKVs int, al
 func (recovery *Recovery) getTotalRegions() int {
 	// Group region peer info by region id.
 	var regions = make(map[uint64][]Regions, 0)
```

Comment:
```suggestion
	var regions = make(map[uint64]struct{}, 0)
```

no need to store region peers here